tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -433,10 +443,26 @@ pub(crate) async fn modify_image(
         .set_attribute(Some(
             ImageAttributeName::LaunchPermission.as_ref().to_string(),
         ))
-        .set_user_ids(Some(modify_opts.user_ids.clone()))
-        .set_user_groups(Some(modify_opts.group_names.clone()))
-        .set_organization_arns(Some(modify_opts.organization_arns.clone()))
-        .set_organizational_unit_arns(Some(modify_opts.organizational_unit_arns.clone()))
+        .set_user_ids(if modify_opts.user_ids.is_empty() {
+            None
+        } else {
+            Some(modify_opts.user_ids.clone())
```

Comment:
@bcressey had made this suggestion to me privately:

```rust
.set_user_ids((!modify_opts.user_ids.is_empty()).then_some(modify_opts.user_ids.clone()))
.set_group_names((!modify_opts.group_names.is_empty()).then_some(modify_opts.group_names.clone()))
```

Might be abit cleaner!

Reply:
I thought `then_some` was unstable. But I'll try it out.

Reply:
Ah bummer. Probably best to avoid unstable then.